Maple2.Server.Core/Packets/Helper/ErrorParserHelper.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Diagnostics;
```

Comment:
Can we put this in Maple2.Tools?

Reply:
nah, it uses stuff from core and core isn't references in tools

Reply:
okay let's put it under Util/Helpers then?

Reply:
didn't notice it was inside /packets lul